docs/toolhive/guides-vmcp/authentication.mdx

```diff
@@ -0,0 +1,159 @@
+---
+title: Authentication
+description: Configure client and backend authentication for Virtual MCP Server.
+---
+
+Virtual MCP implements a two-boundary authentication model that separates client
+authentication from backend authentication, giving you centralized control over
+access while supporting diverse backend requirements.
+
+## Two-boundary authentication model
+
+```mermaid
+flowchart LR
+    subgraph Boundary1[" "]
+        direction TB
+        Client[MCP Client]
+        B1Label["**Boundary 1**<br>Client → Virtual MCP"]
+    end
+
+    subgraph vMCP[Virtual MCP Server]
+        Auth[Token Validation]
+        Exchange[Token Exchange]
```

Comment:
I think we should use token exchange as an example, but the way I read the diagram is that token exchange is the only back end authentication available.

Reply:
Fixed in 2cda252 - removed pass_through auth type, diagram now shows only the implemented auth strategies (discovered, external_auth_config_ref)